rma/controllers/main.py

```diff
@@ -27,10 +28,10 @@ def _rma_get_page_view_values(self, rma, access_token, **kwargs):
     @http.route(['/my/rmas', '/my/rmas/page/<int:page>'],
                 type='http', auth="user", website=True)
     def portal_my_rmas(self, page=1, date_begin=None, date_end=None,
-                       sortby=None, **kw):
+                       sortby=None, filter_domain=None, **kw):
```

Comment:
This is a very bad security practice, as here people can inject things, so please avoid this in a public route. You can have a private method that accepts a domain and do some of the common process, and multiple routes that call this private method.

Remember that "safe_evaling" something has to be done always in private. Anyway, why the need of `safe_eval`? I suppose that is due to the route that passes as string. You remove also this need doing the way I told you.

Reply:
How is this done in other core modules?